rules-tests/TypeDeclaration/Rector/ClassMethod/AddReturnTypeDeclarationRector/Fixture/my_form_type.php.inc

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Symfony\Component\Form;
+
+interface FormTypeInterface
+{
+    /**
+     * @return string|null
+     */
+    public function getParent();
+}
```

Comment:
You can move interface to `Source` directory https://github.com/rectorphp/rector-src/tree/main/rules-tests/TypeDeclaration/Rector/ClassMethod/AddReturnTypeDeclarationRector/Source